src/gui/modalMenu.h

```diff
@@ -77,7 +77,7 @@ class GUIModalMenu : public gui::IGUIElement
 	std::string m_jni_field_name;
 #endif
 #ifdef HAVE_TOUCHSCREENGUI
-	v2s32 m_down_pos;
+	bool m_second_touch = false;
```

Comment:
```suggestion
	// This is set to true if the menu is currently processing a second touch
	// event.
	bool m_second_touch = false;
```

I think this deserves a comment.